charts/spire/charts/spire-server/values.yaml

```diff
@@ -498,13 +505,20 @@ ingress:
     # nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
     # nginx.ingress.kubernetes.io/ssl-passthrough: "true"
 
-  ## @param ingress.hosts [array] Host paths for ingress object
-  hosts:
-    - host: spire-server.example.org
-      paths:
-        - path: /
-          pathType: Prefix
-  ## @param ingress.tls [array] Secrets containining TLS certs to enable https on ingress
+  ## @param ingress.host Host to use for autogenerated rules. If no '.' is in host, trustDomain is automatically appended.
+  host: "spire-server"
+
+  ## @param ingress.tlsSecret secret to use for autogenerated rules.
+  tlsSecret: ""
```

Comment:
Show this have a default value?

Reply:
If specified it needs to exist. not knowing what the user will have loaded, its best probably left unset.

At least in the case of ingress-nginx, it does get the ingress controllers default cert if not specified as well.